examples/tutorials/Part 11 bis - Encrypted inference on ResNet-18.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<ul><li>Where a shared input [[x]] is applied on a public function --&gt; I think it should be the other way around</li><li>alpha is called a random mask (when I hear mask I always tend to consider it like an "index-value" selector)? - maybe a random number?</li><li>x = y + 1 shouldn't continue on the previous line?</li><li>[[f_alpha]]_j (maybe specify that j is in {0, 1}) or explicitly specify [[f_alpha]]_0 and [[f_alpha]]_1 (also, thinking of it, those function shares, can be owned by some other workers, right?)</li><li>Maybe: moving the "function shares [[f_alpha]] is also shared between Alice and Bob" where you give the example</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>OK I 've tried to improve!</p><p> </p><ul><li>Where a shared input [[x]] is applied on a public function --&gt; I think it should be the other way around. --&gt; I wouldn keep this for secureNN because they both to the same "public" protocol to evaluate the function, and have a secret shared input</li></ul><p> </p>